packages/mgt-chat/src/components/Chat/Chat.tsx

```diff
@@ -96,8 +109,24 @@ export const Chat = ({ chatId }: IMgtChatProps) => {
             </>
           ) : (
             <>
-              {chatState.status}
-              <Spinner />
+              {isLoading && (
+                <div className={styles.spinner}>
+                  <Spinner /> <br />
+                  {chatState.status}
+                </div>
+              )}
+              {chatState.status === 'initial' && (
+                <ChatMessageBar messageBarType={MessageBarType.info} message={'Select a chat to display messages.'} />
```

Comment:
I wouldn't invite users to "select" a chat as it might not be valid in some scenarios. As it's in the initial state, I don't even think we need a rendering here. Can we expand on these scenarios?